framework/templates/ecl-templates-search/ecl-templates-search.config.js

```diff
@@ -1,12 +1,16 @@
 module.exports = {
   title: 'Search Page',
   label: 'Search Page',
-  status: 'wip',
+  status: 'ready',
   preview: '@preview-full-page',
   tags: ['template'],
   context: {
     global: {
       language: 'en',
     },
+    _demo: {
+      scripts:
```

Comment:
Another tiny detail here is that in `General Listing Mobile Filters Displayed.png` the button for collapsing the filters is shown as a "Hide" link